Jint/Runtime/Interop/ObjectWrapper.cs

```diff
@@ -98,6 +98,16 @@ public override JsValue Get(JsValue property, JsValue receiver)
                 return Undefined;
             }
 
+            if (Target is IDictionary<string, object> stringKeyedDictionary && property is JsString dicKey) // expando object for instance
```

Comment:
this could also be part of next if also where we know it's a JsString and also `ToString` will be done? No sure if we could be preventing registerin custom ExpandObject accessor now in engine, but probably this is anyway what end user is after.

Reply:
Done